presto-docs/src/main/sphinx/release/release-325.rst

```diff
@@ -28,18 +31,18 @@ Web UI Changes
 Hive Changes
 ------------
 
-* Fix incorrect behavior of ``CREATE TABLE`` when Hive metastore is configured
+* Fix incorrect behavior of ``CREATE TABLE``, when Hive metastore is configured
```

Comment:
please split this out of "Add release date .." commit to a separate

Reply:
Is it worth doing that for the few simple commas?